accounts-db/src/tiered_storage.rs

```diff
@@ -302,4 +306,137 @@ mod tests {
         // expect the file does not exist as the file has been removed on drop
         assert!(!tiered_storage_path.try_exists().unwrap());
     }
+
+    /// Create a test account based on the specified seed.
+    /// The created test account might have default rent_epoch
+    /// and write_version.
+    fn create_test_account(seed: u64) -> (StoredMeta, AccountSharedData) {
+        let data_byte = (seed % 256) as u8;
+        let account = Account {
+            lamports: seed,
+            data: (0..seed as usize).map(|_| data_byte).collect(),
+            owner: Pubkey::new_unique(),
+            executable: seed % 2 > 0,
+            rent_epoch: if seed % 3 > 0 { seed } else { u64::MAX },
+        };
+
+        let stored_meta = StoredMeta {
+            write_version_obsolete: if seed % 5 > 0 { seed } else { u64::MAX },
+            pubkey: Pubkey::new_unique(),
+            data_len: seed,
+        };
+        (stored_meta, AccountSharedData::from(account))
+    }
+
+    /// The helper function for all write_accounts tests.
+    /// Currently only supports hot accounts.
+    fn write_accounts_test_impl(
+        path_suffix: &str,
+        account_data_sizes: &[u64],
+        format: TieredStorageFormat,
+    ) {
+        let accounts: Vec<(StoredMeta, AccountSharedData)> = account_data_sizes
```

Comment:
`Vec<_>` here should be sufficient here, but given it is a tuple, I think using `Vec<(StoredMeta, AccountSharedData)>` explicitly can improve the readability.

Let me know if you prefer `Vec<_>` here.

Reply:
Done